client/src/rpb/index.js

```diff
@@ -345,7 +345,8 @@ class RPB extends React.Component {
                     }, 200);
                   }
                 }}
-                titleId="tbp-title"
```

Comment:
The corresponding element with `id="tbp-title"` doesn't need that id anymore, might as well clean that up too 👍